packages/core/src/Stores/client-store.js

```diff
@@ -1272,7 +1287,6 @@ export default class ClientStore extends BaseStore {
             if (this.is_logged_in && !has_client_accounts) {
                 localStorage.setItem(storage_key, JSON.stringify(this.accounts));
                 LocalStore.set(storage_key, JSON.stringify(this.accounts));
-                this.syncWithLegacyPlatforms(client_id, this.accounts);
```

Comment:
why have we removed this? Not seeing any indication that we want to remove this only for DerivEz. I understand derivEz accounts should not be available for smarttrader or binarybot. But i think removing this may break platform sync for non derivEz accounts between DTrader, Dbot, SmartTrader, and binarybot.